index.js

```diff
@@ -429,13 +385,11 @@ export function StateT(M) {
   //. Z.of (Maybe, 2)
   //. ```
   StateT.prototype['fantasy-land/chain'] = function(f) {
-    var self = this;
-    return new StateT (function(s) {
-      return Z.chain (
-        function(state) { return f (state.value).run (state.state); },
-        self.run (s)
-      );
-    });
+    const self = this;
+    return new StateT (s => Z.chain (
+      state => f (state.value).run (state.state),
+      self.run (s)
+    ));
```

Comment:
```suggestion
    return new StateT (s => Z.chain (
      state => f (state.value).run (state.state),
      this.run (s)
    ));
```

What about this one?

Reply:
Didn't see that one. I will review all of them this time :)

Reply:
Somehow I also missed this one last time.